WireDomain/Sources/WireDomain/Synchronization/PullAllFeatureConfigsSync.swift

```diff
@@ -120,9 +120,6 @@ extension FeatureConfigLocalStoreProtocol {
                 config: config.toDomainModel()
             )
         case let .consumableNotifications(config):
-            // Necessary to log correct sync version
-            LogAttributes.consumableNotificationsEnabled = config.status == .enabled
```

Comment:
this part was a bit flaky as sync objects were built before we fetch feature config. I decided to remove it and specify the right version wherever I could direcly